src/index.ts

```diff
@@ -1,11 +1,49 @@
 import { Probot } from "probot";
 
 export = (app: Probot) => {
+  
+  // Respond to an issue opened 
   app.on("issues.opened", async (context) => {
     const issueComment = context.issue({
       body: "Thanks for opening this issue!",
     });
     await context.octokit.issues.createComment(issueComment);
+  });  
+  // Respond to the GitHub app installation 
+  app.on("installation.created", async (context) =>{
+
+    // TODO: get the installation token from the url
+    const installationTokenUrl = context.payload.installation.access_tokens_url;
+    const installationToken = await context.octokit.request("Get {url}",{url: installationTokenUrl});
+    app.log.info("installationt token: {}",installationToken);
+    // TODO: iterate over the list of repositories
+    const repos = context.payload.repositories;
+    const orgName = repos[0].full_name.split("/")[0];
+
+    const k8s = require('@kubernetes/client-node');
+    const kc = new k8s.KubeConfi();
+    kc.loadFromDefault();
+    const k8sApi = kc.makeApiClient(k8s.coreV1Api);
+    const k8sContext = kc.getCurrentContext();
+    const namespace = k8sContext.split('/')[0];
+
+    var secret = {
+        metadata :{
+          name : "<installation_id>",
```

Comment:
The name here should be `context.payload.installation.id` I think. :slightly_smiling_face: